projects/demo/src/modules/pipes/stringify/examples/1/index.ts

```diff
@@ -1,14 +1,30 @@
 import {Component} from '@angular/core';
+import {FormsModule} from '@angular/forms';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {
+    TuiComboBoxModule,
+    TuiDataListWrapperModule,
+    TuiFilterByInputPipeModule,
+    TuiStringifyContentPipeModule,
+    TuiStringifyPipe,
+} from '@taiga-ui/kit';
 
 @Component({
-    selector: 'tui-stringify-example1',
+    standalone: true,
+    imports: [
+        TuiComboBoxModule,
+        FormsModule,
```

Comment:
maybe to put Angular modules in the first place, and then taiga?

```suggestion
        FormsModule,
        TuiComboBoxModule,
```